### PR TITLE
add policy regarding policy repository

### PR DESCRIPTION
**Pull request by larsks**

### CODE REVIEW

File: specs/spec-repository.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# MOC Operational Policy Repository
+
+This describes our approach to documenting MOC operational policy and
+procedures.
+
+## Problem Description
+
+We want to establish various policies around the management of our
+infrastructure, but we don't have an established place to put such
+documentation.
```
Comment (knikolla) [inline specs/spec-repository.md:10]: So just to clarify, this repository is just about the operational policies, and not about operational documentation. Makes sense to me, and this sort of policy documentation is something that we are lacking.

**Review by knikolla — APPROVED** [review]
Looks good to me. Approve of the direction and objectives.

File: specs/spec-repository.md (hunk 1 above)
Comment (knikolla) [reply specs/spec-repository.md:10]: I guess the wording through me a bit off. I'm assuming this we are on right now is the operation policy repository. And this is also the operational docs repository.

Comment (larsks) [reply specs/spec-repository.md:10]: I was originally thinking of it as primarily for policy documents (like this one). It sounds like we are on the same page on that point?

Comment (naved001) [reply specs/spec-repository.md:10]: I am confused. I thought this current repository is the one holding operational policy documentation. What would be the purpose of a different repository and the distinction between the two?

**Pull request merged**